### PR TITLE
Add setup.py for packaging and running tests

### PR DESCRIPTION
**Pull request by petroav**
Task: SWARM-1121
Testing: make dist test

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.PHONY: clean-pyc clean-build clean-test test clean
```
Comment (anusudarsan) [inline Makefile]: add copyright? here and below

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+test_requirements = [
+    'tox==1.9.2',
+    'nose==1.3.7',
+    'mock==1.0.1',
+]
+
+
+setup(
+    name='presto',
```
Comment (anusudarsan) [inline setup.py]: should we name it as ambari-presto or sth like that?

**Comment by anusudarsan** [comment]
lgtm mod comments. Do you have a README yet? if so, it will be nice to update the build/testing process there

File: setup.py (hunk 2 above)
Comment (petroav) [inline setup.py]: Sure, I'll change it.

File: Makefile (hunk 1 above)
Comment (petroav) [inline Makefile]: I'm waiting on @mattsfuller to give me the green light to put Apache licenses on all the files.

**Comment by petroav** [comment]
Thanks for the review! I'll do another PR with all the license stuff in there. I'll also do another PR with a README.

**Pull request closed without merging**